packages/components/docs/sass.md

```diff
@@ -22861,6 +22861,14 @@ Tile styles
     }
   }
 
+  .#{$prefix}--tile--expandable {
+    width: inherit;
```

Comment:
Honestly I think there is not many use cases for `width: inherit`. So may I ask what this is for?

Reply:
I just wanted it to get the width that was already defined in `bx--tile`, but I see that doing `width: 100%` gets the same result so I'll change it to that!